_assets/ci/Jenkinsfile.ios

```
@@ -47,6 +47,9 @@ pipeline {
     )
     /* fix for gomobile complaining about missing packages */
     CGO_ENABLED = "1"
+    /* Use `go generate`, ignore cache on CI. 
+    Also, `go-generate-fast` was struggling with some paths access and didn't work anyway */
+    GO_GENERATE_FAST = "false"
```

Way too fucking verbose. People don't read long comments:
```suggestion
    /* Generate files without cache to avoid weird bugs. */
    GO_GEN_CMD = 'go generate'
```

---

Fixed: ac38f50409b761d74709068f4b08634eeb5f631d